test/integration/e2e_test.py

```diff
@@ -698,9 +698,8 @@ def test_in_sequence(self):
 
         self.client.new_channel(SYSTEM_CHANNEL_NAME)
 
-        loop.run_until_complete(self.get_channel_config_with_orderer())
-
         loop.run_until_complete(self.channel_create())
+        time.sleep(5)
```

Comment:
also why there is a sleep added here?

Reply:
In the raft consensus, the channel creation takes some time and I was getting errors so I added the sleep function.